src/main/java/cc/hyperium/mods/chunkanimator/ChunkAnimatorConfig.java

```diff
@@ -0,0 +1,33 @@
+package cc.hyperium.mods.chunkanimator;
+
+/*
+ * Created by Cubxity on 12/1/2018
+ */
+
+import cc.hyperium.config.*;
+
+import static cc.hyperium.config.Category.CHUNK_ANIMATOR;
+
+public class ChunkAnimatorConfig {
+    @ConfigOpt
+    @ToggleSetting(name = "Enable", mods = true, category = CHUNK_ANIMATOR)
+    public static boolean enabled = false;
+
+    @ConfigOpt
```

Comment:
https://github.com/HyperiumClient/Hyperium/blob/f83b42692ccefbfc4c0cd84754ecdae9ebfbb44b/src/main/java/cc/hyperium/mods/chunkanimator/ChunkAnimatorConfig.java#L16-L24

What does this "mode" mean? An explanation of what the modes correspond to will improve user experience.

Reply:
I'm not exactly sure which mode does what, since the original mod doesn't have very detailed documentation.

Reply:
I'll build your branch locally and document them later when I get home.